app/components/alert_component.rb

```diff
@@ -1,12 +1,12 @@
 # frozen_string_literal: true
 
 class AlertComponent < BaseComponent
-  VALID_TYPES = %i[info success warning error emergency other].freeze
+  VALID_TYPES = %i[info success warning error emergency].freeze
```

Comment:
Alternatively, I could...

```suggestion
  VALID_TYPES = [nil, :info, :success, :warning, :error, :emergency].freeze
```

...to avoid the validation changes below.

No strong preference from me 🤷

Reply:
I like this suggestion to keep the initializer simpler

Reply:
Updated in 001f103f82.